homeassistant/components/mobile_app/webhook.py

```diff
@@ -312,8 +312,8 @@ async def webhook_render_template(hass, config_entry, data):
 @validate_schema(
     {
         vol.Optional(ATTR_LOCATION_NAME): cv.string,
-        vol.Required(ATTR_GPS): cv.gps,
-        vol.Required(ATTR_GPS_ACCURACY): cv.positive_int,
+        vol.Optional(ATTR_GPS): cv.gps,
+        vol.Optional(ATTR_GPS_ACCURACY): cv.positive_int,
```

Comment:
This also drops the requirement of these two to be reported always in pairs. Should we adjust the current change to keep that requirement?

So, if a `gps` is provided, a `gps_accuracy` is also required?

Reply:
Makes sense. I'll add this.